docs/guides/src/main/server/run-containers.adoc

```diff
@@ -0,0 +1,106 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Running in a container"
+summary="Learn how to run Keycloak from a container image"
+priority=20
+includedOptions="">
```

Comment:
Should we include some options here? Standard stuff that is needed when running a container? Or, at least include the options that you've used in the examples?

Reply:
I can do this, but might grow quite big. Let me try. Btw: What about KEYCLOAK_ADMIN / KEYCLOAK_ADMIN_PASSWORD - they're not documented yet in the all-config guide, and are kinda special.